src/mlpack/methods/naive_bayes/nbc_main.cpp

```diff
@@ -123,9 +125,10 @@ static void mlpackMain()
   RequireOnlyOnePassed({ "training", "input_model" }, true);
   ReportIgnoredParam({{ "training", false }}, "labels");
   ReportIgnoredParam({{ "training", false }}, "incremental_variance");
-  RequireAtLeastOnePassed({ "output", "output_model", "output_probs" }, false,
+  RequireAtLeastOnePassed({ "output","predictions","output_model", "output_probs" }, false,
```

Comment:
Missing space right after the comma.

Reply:
Corrected please take a look at it again .